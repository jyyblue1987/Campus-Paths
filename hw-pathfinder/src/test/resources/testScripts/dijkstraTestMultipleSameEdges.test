# Find a path with Dijkstra's algorithm when there are multiple edges with the same start and end node

CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B 0.5
AddEdge graph A B 2.0
AddEdge graph A B 3.0
AddEdge graph A B 1.0
AddEdge graph A B 5.0

FindPath graph A B