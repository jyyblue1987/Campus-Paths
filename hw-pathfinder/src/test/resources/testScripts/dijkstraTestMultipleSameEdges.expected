# Find a path with Dijkstra's algorithm when there are multiple edges with the same start and end node

created graph graph

added node A to graph
added node B to graph

added edge 0.500 from A to B in graph
added edge 2.000 from A to B in graph
added edge 3.000 from A to B in graph
added edge 1.000 from A to B in graph
added edge 5.000 from A to B in graph

path from A to B:
A to B with weight 0.500
total cost: 0.500